controllers/account-setting/my-team/get-my-agents.controller.js

```diff
@@ -4,6 +4,14 @@ const { getAgentsListForMyTeam } = require('../../../services/myteam.service');
 const getMyAgentsController = async (req, res, next) => {
 	try {
 		const user = req.user;
+
+		if (user.role !== 'CRM_ADMIN') {
```

Comment:
This is redundant and bad practise. use a middleware. Already implemented with routes in notifications or auth